docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -5,101 +5,123 @@
 
-After you save your module file in one of these locations, Ansible loads it and you can use it in any local task, playbook, or role.
+* Add your standalone local module to one of the default configured locations. Read the documentation for the :ref:`DEFAULT_MODULE_PATH` configuration setting for details. Default locations may change without notice.
+* Add the location of your standalone local module to an environment variable or configuration:
+   * the ``ANSIBLE_LIBRARY`` environment variable
+   * the :ref:`DEFAULT_MODULE_PATH` configuration setting
+
+To view your current configuration settings for modules:
+
+.. code-block:: text
 
-To confirm that ``my_custom_module`` is available:
+   ansible-config dump |grep DEFAULT_MODULE_PATH
 
-* type ``ansible localhost -m my_custom_module``. You should see the output for that module.
+After you save your module file in one of these locations, Ansible loads it and you can use it in any local task, playbook, or role.
 
-or
+To confirm that ``my_local_module`` is available:
 
-* type ``ansible-doc -t module my_custom_module``. You should see the documentation for that module.
+* type ``ansible localhost -m my_local_module`` to see the output for that module, or
+* type ``ansible-doc -t module my_local_module`` to see the documentation for that module
 
 .. note::
 
    Currently, the ``ansible-doc`` command can parse module documentation only from modules written in Python. If you have a module written in a programming language other than Python, please write the documentation in a Python file adjacent to the module file.
 
-You can limit the availability of your local module. If you want to use a local module only with selected playbooks or only with a single role, load it in one of the following locations:
+Adding standalone local modules for selected playbooks or a single role
+-----------------------------------------------------------------------
+
+Ansible automatically loads all executable files from certain directories adjacent to your playbook or role as modules. Standalone modules in these locations are available only to the specific playbook, playbooks, or role in the parent directory.
 
-* In a selected playbook or playbooks: Store the module in a subdirectory called ``library`` in the directory that contains those playbooks.
-* In a single role: Store the module in a subdirectory called ``library`` within that role.
+* To use a standalone module only in a selected playbook or playbooks, store the module in a subdirectory called ``library`` in the directory that contains the playbook or playbooks.
+* To use a standalone module only in a single role, store the module in a subdirectory called ``library`` within that role. If you are developing new modules, we recommend distributing them in :ref:`collections <developing_collections>`, not in roles.
 
```

Comment:
Is it worth mentioning here that for roles in a collection, you can no longer include a standalone module as part of that role?

Reply:
Ah, good catch, yes.

Reply:
addressed in d1103127ab512e487cd3dd6ec84baa420def0d3f